src/plugincode/scan_proper_hooks.py

```diff
@@ -39,3 +42,6 @@ def add_cmdline_option(scan_output_plugins):
     Return a click.Option instance which will be added to scancode.cli.ScanCommand
     """
     pass
+
+scan_proper_plugins = PluginManager('scan_proper')
+scan_proper_plugins.add_hookspecs(sys.modules[__name__])
```

Comment:
Actually, what did you mean by being explicit about the hookspec? I mean `add_hookspecs` method does not take functions so a module needs to be passed. 
And since the current module is the one containing hookspecs, it needs to be passed.